### DISCUSSION TITLE
Building studio is very slow

### DISCUSSION THREAD

**Discussion by wrux** [sanity-io/sanity#3369] [Sanity Studio Support]
I have an empty studio instance which starts up in less than a second, but somehow takes a few minutes to build. When adding some schema and a plugin or 2, I'm looking at 10+ minute builds.

A meaningful build output would be nice as it's not clear what's taking so long. Perhaps a flag for verbose mode?

**Comment by p10e** [comment]
Hi @wrux - that seems slower than expected indeed. Could you try prefixing the build command with `DEBUG=sanity*` or even `DEBUG=*` to get further insights as to why it might be taking so long?

For example:
```DEBUG=sanity* npx sanity build```

**Reply by susickypavel** [reply]
I'm finding builds of Sanity Studio v3 very slow. As OP said, development is done under a second on an almost empty studio using Vite, but building on M1 Mac takes nearly one minute to complete.

I've used the DEBUG variable to see what's going on, and with DEBUG=*, it seems like the 99% of build time is eaten by babel transpiling the code as it logs quadrillion lines from the build start to the end. Then the question is whether the transpilation should take so long or not?

Knowing that it still uses babel to transpile the code, will this change in the future release of v3?
